content/source/docs/enterprise/before-installing/reference-architecture/aws.md

```diff
@@ -345,6 +345,8 @@ primary AWS Region hosting the Terraform Enterprise application failing, the sec
 AWS Region will require some configuration before traffic is directed to
 it along with some global services such as DNS.
 
+Please note that with _Standalone_ implementation mode, only one Terraform Enterprise instance can be running against the same database. This deployment acts to minimize the Mean Time To Recovery (MTTR) in the event of a regional failure, avoiding the need to replicate and stand up the data plane infrastructure during an outage.
```

Comment:
I think it might be helpful to move this up so that it comes right after the warm and cold standby sentence. Right now, it interrupts the flow because it comes in between the sentence about configuration and the following examples.